lib/l10n/intl_en.arb

```diff
@@ -78,6 +83,8 @@
   "hintEvaluation": "Final exam",
   "hintPoints": "4.0",
 
+  "helperContactInfo": "In case we need to reach out for more info",
```

Comment:
To me, it sounds good, but I'm not sure if the regular user would understand the meaning of the sentence. And what would be the equivalent for 'ro' language? Or do we keep the same one that I used?

Reply:
keep the ro one. Also the user better know English if they have the app in English xD

Reply:
I just realised - rather than having an entire category just for this (I guess you noticed the first word in the name is a category), let's move it to the info category. "infoContactInfo" is fine.